docs/user-guide/text-editor-toolbar.md

```diff
@@ -1,27 +1,29 @@
 
-* **Text style** to insert text in *Bold* (<img src="../img/button/bold-button.jpg" class="ms-docbutton"/>), *Italic* (<img src="../img/button/italic-button.jpg" class="ms-docbutton"/>) or *Underline* (<img src="../img/button/underline_button.jpg" class="ms-docbutton"/>) 
+* **Text style** to insert text in *Bold* <img src="../img/button/bold.jpg" class="ms-docbutton" style="max-height:15px;"/>, *Italic* <img src="../img/button/italic.jpg" class="ms-docbutton" style="max-height:15px;"/>, *Underline* <img src="../img/button/underline.jpg" class="ms-docbutton" style="max-height:15px;"/> or *Strikethrough* <img src="../img/button/strike.jpg" class="ms-docbutton" style="max-height:15px;"/>
 
-* **Block Quotation** to insert a quote in the text (<img src="../img/button/quote-button.jpg" class="ms-docbutton"/>)
+* **Monospace** <img src="../img/button/monospace.jpg" class="ms-docbutton" style="max-height:15px;"/> to insert the same space between words
 
-* **Bullet list** to create a list (<img src="../img/button/bullet-button.jpg" class="ms-docbutton"/>)
+* **Alignment** <img src="../img/button/aligne.jpg" class="ms-docbutton" style="max-height:15px;"/> inside the text window (*Left*, *Center*, *Right* or *Justify*)
 
-* **Alignment** inside the text window (*Left*, *Center*, *Right* or *Justify*) (<img src="../img/button/aligne-button.jpg" class="ms-docbutton"/>)
+* **Color Picker** <img src="../img/button/color-picker.jpg" class="ms-docbutton" style="max-height:15px;"/> to change the text color
 
-* **Color Picker** to change the text color (<img src="../img/button/color-text-button.jpg" class="ms-docbutton"/>), customize the highlighting (<img src="../img/button/hightlight-button.jpg" class="ms-docbutton"/>) or clean the color (<img src="../img/button/clean-button.jpg" class="ms-docbutton"/>)
+* **Bullet list** to create a *Unordered* list <img src="../img/button/unordered.jpg" class="ms-docbutton" style="max-height:15px;"/> or *Ordered* list <img src="../img/button/ordered.jpg" class="ms-docbutton" style="max-height:15px;"/>
 
-* **Add image** to add an image on the text panel (<img src="../img/button/image-button.jpg" class="ms-docbutton"/>)
+* **Indent/Outdent** <img src="../img/button/indent.jpg" class="ms-docbutton" style="max-height:15px;"/> to  indent the text in relation to the left margin or to the right margin
 
-* **Link** to convert in hyperlink the selected portion of text (<img src="../img/button/link-button.jpg" class="ms-docbutton"/>)
+* **Link** <img src="../img/button/link.jpg" class="ms-docbutton" style="max-height:15px;"/> to convert in hyperlink the selected portion of text
+
+* **Remove** <img src="../img/button/remove4.jpg" class="ms-docbutton" style="max-height:15px;"/> to remove the formatting 
 
 !!! note
     In order to connect to a website external to [Mapstore](https://mapstore.geo-solutions.it/mapstore/#/), the protocol must be specified (e.g., `http://` or `https://`).
```

Comment:
In order to setup an hyperlink to an external website, the protocol must be specified (e.g., `http://` or `https://`).